releasenotes/notes/0.18/dynamical-decoupling-3ce6e882954f6d3b.yaml

```diff
@@ -13,7 +13,7 @@ features:
         from qiskit.circuit import QuantumCircuit
         from qiskit.circuit.library import XGate
         from qiskit.transpiler import PassManager, InstructionDurations
-        from qiskit.transpiler.passes import ALAPSchedule, DynamicalDecoupling
+        from qiskit.transpiler.passes import ALAPSchedule, DynamicalDecoupling, PadDelay
```

Comment:
Seems like updating past release note is bit weird though this is TEMPORARY change. Since DD pass can be a subclass of `BasePadding` pass (i.e. it inserts DD sequence instead of long delay), we can in principle skip `PadDelay` here after the update. I avoided the upgrade of DD pass since it is not within the scope of this PR, but I lean toward updating DD pass too in this PR, at the price of extra review overhead.

Reply:
Adding the padding pass to the requirements of DD solved this issue. I think DD pass is not used in standalone thus this code is okey (anyways this is temporary).